driver/README.md

```diff
@@ -14,7 +14,9 @@ Then run:
 ```bash
```

Comment:
This is actually somewhat outdated since we build the driver image as part of the Hail update workflow now and probably should never push a new image manually.

How about changing lines 11 and below to:

```markdown
The driver image gets rebuilt and pushed automatically as part of the [Hail update workflow](../.github/workflows/hail_update.yaml).
```

Reply:
Yeah I completely agree, I thought we might've done this, but couldn't remember.